### ISSUE TITLE
Empty ReferenceSourceTarget added to project

### ISSUE THREAD

**Issue by mrward** [mono/monodevelop#7573]
Adding a project reference to a project will result in an empty ReferenceSourceTarget element being added.

```
<Project Sdk="Microsoft.NET.Sdk">

  <PropertyGroup>
    <OutputType>Exe</OutputType>
    <TargetFramework>netcoreapp2.2</TargetFramework>
  </PropertyGroup>

  <ItemGroup>
    <ProjectReference Include="..\Lib\Lib.csproj">
      <ReferenceSourceTarget></ReferenceSourceTarget>
    </ProjectReference>
  </ItemGroup>
</Project>

```

This is likely to be due to the new ItemDefinitionGroup support.

> VS bug [#893190](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/893190)

**Comment by xamarin-release-manager** [comment]
> [sync] [VS-10] Comment by **xamarinc**



Fixed in version 8.2.0.332 (master)



Pull Request #7583 merged by: Lluis Sanchez  
 Author: mono  
 Commit: fa7f6b5592402e11422ae192c3308dcf4cbb9323 (mono/monodevelop)

**Comment by xamarin-release-manager** [comment]
> [sync] [VS-14] Comment by **xamarinc**



Fixed in version 8.6.0.802 (master)



Pull Request #7583 merged by: Lluis Sanchez  
 Author: mono  
 Commit: fa7f6b5592402e11422ae192c3308dcf4cbb9323 (xamarin/monodevelop)
